homeassistant/components/tuya/light.py

```diff
@@ -123,6 +123,11 @@ class TuyaLightEntityDescription(LightEntityDescription):
             color_temp=DPCode.TEMP_VALUE,
             color_data=DPCode.COLOUR_DATA,
         ),
+        # Some ceiling fan lights use LIGHT for DPCode instead of SWITCH_LED
+        TuyaLightEntityDescription(
+            key=DPCode.LIGHT,
+            name="Light",
```

Comment:
The name should not be needed (just like the one above).

```suggestion
```

Reply:
If I don't put the name, it names the light toggle whatever you have the device named in the tuya app. Setting the name allows you to differentiate between the fan control and the light control. The 'dj' device above this one does something similar so that's why I thought this would be alright as well.

Reply:
This is a single primary `light` entity, providing it with a name (following the newer latest standards), should not be done. Hence it should not be added here.

Reply:
Fixed